tests/testdata/velociraptor/Windows.Forensics.ProcessInfo.json

```diff
@@ -0,0 +1,10 @@
+{"Name":"System","PebBaseAddress":"0x0","Pid":4,"ImagePathName":null,"CommandLine":null,"CurrentDirectory":null,"Env":{}}
+{"Name":"smss.exe","PebBaseAddress":"0x5b6ea41000","Pid":268,"ImagePathName":null,"CommandLine":null,"CurrentDirectory":null,"Env":{}}
+{"Name":"csrss.exe","PebBaseAddress":"0x91fdbd0000","Pid":348,"ImagePathName":null,"CommandLine":null,"CurrentDirectory":null,"Env":{}}
+{"Name":"wininit.exe","PebBaseAddress":"0xc5f8d8e000","Pid":416,"ImagePathName":null,"CommandLine":null,"CurrentDirectory":null,"Env":{}}
+{"Name":"csrss.exe","PebBaseAddress":"0x65715c4000","Pid":424,"ImagePathName":null,"CommandLine":null,"CurrentDirectory":null,"Env":{}}
+{"Name":"winlogon.exe","PebBaseAddress":"0xaa675e0000","Pid":492,"ImagePathName":"C:\\Windows\\system32\\winlogon.exe","CommandLine":"winlogon.exe","CurrentDirectory":"C:\\Windows\\system32\\","Env":{"ALLUSERSPROFILE":"C:\\ProgramData","ChocolateyInstall":"C:\\ProgramData\\chocolatey","CommonProgramFiles":"C:\\Program Files\\Common Files","CommonProgramFiles(x86)":"C:\\Program Files (x86)\\Common Files","CommonProgramW6432":"C:\\Program Files\\Common Files","COMPUTERNAME":"WKST01","ComSpec":"C:\\Windows\\system32\\cmd.exe","NUMBER_OF_PROCESSORS":"2","OS":"Windows_NT","Path":"C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files\\Amazon\\cfn-bootstrap\\;C:\\ProgramData\\chocolatey\\bin;;C:\\Program Files\\Microsoft VS Code\\bin","PATHEXT":".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC","PROCESSOR_ARCHITECTURE":"AMD64","PROCESSOR_IDENTIFIER":"Intel64 Family 6 Model 85 Stepping 7, GenuineIntel","PROCESSOR_LEVEL":"6","PROCESSOR_REVISION":"5507","ProgramData":"C:\\ProgramData","ProgramFiles":"C:\\Program Files","ProgramFiles(x86)":"C:\\Program Files (x86)","ProgramW6432":"C:\\Program Files","PSModulePath":"%ProgramFiles%\\WindowsPowerShell\\Modules;C:\\Windows\\system32\\WindowsPowerShell\\v1.0\\Modules;C:\\Program Files (x86)\\AWS Tools\\PowerShell\\","PUBLIC":"C:\\Users\\Public","SystemDrive":"C:","SystemRoot":"C:\\Windows","TEMP":"C:\\Windows\\TEMP","TMP":"C:\\Windows\\TEMP","USERNAME":"SYSTEM","USERPROFILE":"C:\\Windows\\system32\\config\\systemprofile","windir":"C:\\Windows"}}
```

Comment:
Going back to my earlier comment about how we identify the right schema for each log file, it looks like several columns would be missed here if we only looked at the columns in the first row.

Reply:
Not sure that they are. The big thing at the end is the contents of the current environment (although it is itself a json dictionary - not sure I want to cover all eventualities of what might be in a nested field. Most data providers don't populate the schema so I've always considered it a bit of nice addition rather than core functionality.